config/routes.rb

```diff
@@ -20,4 +20,7 @@
       get :star
     end
   end
+
+  get '/user_preferences', to: 'users#edit'
```

Comment:
Settings is fine with me.  My main reason for changing it from /edit was to avoid ambiguity about what was being edited.